src/Explorer/Controls/Settings/SettingsSubComponents/SubSettingsComponent.tsx

```diff
@@ -207,6 +207,7 @@ export class SubSettingsComponent extends React.Component<SubSettingsComponentPr
             value={this.props.timeToLiveSeconds?.toString()}
             onChange={this.onTimeToLiveSecondsChange}
             suffix="second(s)"
+            ariaLabel={`Time to live on ${this.props.timeToLiveSeconds} seconds`}
```

Comment:
I'm not sure what the intent is here but "Time to live on x seconds" doesn't make sense to me. A more grammatically correct sentence would be "Time to live is x seconds." However, I don't think the purpose of aria-label is to provide a name for the current control, i.e. its purpose rather than the value of it as I believe a screen reader reads the value. So my recommendation for this property would be "Time to live in seconds" without any dynamic value being read as part of its name.